src/main/webapp/user-private-comments.jsp

```diff
@@ -14,27 +13,53 @@
 %>
 
 <%--HTML--%>
-<ul>
-  <%if (currUser) {%>
-    <h1 id="pc-header">Your Private Task Comments:</h1>
-    <%for (TaskData task : tasks) {%>
-      <li class="task-with-pc">
-        <button class="inline deep-button" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
-          <%=task.getName()%>
-        </button>
-        <%request.setAttribute("task", task);%>
-        <%request.setAttribute("clickable", false);%>
-        <jsp:include page="task-status-checkmark.jsp"/>
-        <p class="inline"><%=task.getDescription()%></p>
-        <form action="/user-profile" method="POST">
-          <input type="hidden" name="taskID" value="<%=task.getTaskID()%>">
-          <br>
-          <textarea style="width:300px; height:100px;" type="text" name="message-<%=task.getTaskID()%>" rows="6" cols="50"><%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%></textarea>
-          <br>
-          <button class="deep-button" style="font-size: 12px;" type="submit" class="deep-button">Update Comment</button>
-        </form>
-        <br><br>
-      </li>
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/@editorjs/editorjs@latest"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@editorjs/header@latest"></script>
+</head>   
+  <ul>
+    <%if (currUser) {%>
+      <h1 id="pc-header">Your Private Task Comments:</h1>
+      <%for (TaskData task : tasks) {%>
+        
+        <script>
+          function init() {
+            const editor = new EditorJS({
+            holderId : 'box',
+            autofocus: true,
+            data: '<%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%>',
```

Comment:
this is not working to load from java. Is there a different way to do it? I looked at a couple of examples and this seemed like the right way to pass from java-->js

Reply:
your editorjs setup looks correct I think.  Couple things though:
- `holderId : 'box'`  <- "box" here is the ID of the div you want to convert into an editorjs box.  I'm not seeing where you have a `<div id="box">`
- since you have the setup wrapped in an `init()` function, you'll need to call the function for it to build the editor box
- I see you have the scripts with the init() and save() functions inside the tasks loop.  This will mean it'll create the init() and save() methods multiple times.  JS I think lets you do this (other languages would complain), but it would only keep the last version, meaning the one with the last tasks's info.  What you can do instead is build init() and save() once (probably in the js file), but be flexible enough to be called on for multiple boxes (since you have multiple tasks)